style.css

```diff
@@ -107,14 +106,65 @@ input {
 #addBtn {
   color: white;
   padding: 1% 2%;
-  background-color: rgb(47, 95, 255);
-  margin-left: 20rem;
+  font-size: 14px;
+  margin-left: 25rem;
   border: 1px solid grey;
 }
 
 .remove {
   margin-right: 3%;
-  border: 2px solid rgb(54, 54, 54);
-  padding: 2% 7%;
+  border: 1px solid rgb(54, 54, 54);
+  padding: 0.6% 2%;
   background-color: rgb(141, 136, 136);
 }
+
+.contact {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+}
+
+article {
+  text-align: left;
+}
```

Comment:
[Optional] I suggest you style your wireframe so that the text are align 👍 
![image](https://user-images.githubusercontent.com/66112760/165642105-5fb41e47-c441-4c84-9704-38e195efd6ea.png)